frontend/src/components/admin/AdminLoginForm.js

```diff
@@ -0,0 +1,99 @@
+import { useState, useEffect } from "react";
```

Comment:
this is still a page right?
So transfer it to pages/user/admin

Also transfer the userLogin to pages as well

Reply:
They are components sa page sir :bow: